app/models/requests/service_eligibility/aeon.rb

```diff
@@ -4,21 +4,31 @@ module ServiceEligibility
     # This class is responsible for determining if a specific
     # resource can be requested via Aeon
     class Aeon
-      def initialize(requestable:)
+      def initialize(requestable:, patron:)
         @requestable = requestable
+        @patron = patron
       end
 
       def to_s
         'aeon'
       end
 
       def eligible?
+        return false unless patron_group_eligible? || patron.guest?
```

Comment:
Do we need to add this check for Aeon items? All users should be able to see this option. I think https://github.com/pulibrary/orangelight/blob/main/docs/requests/patron_groups.md is incorrect. Aeon should probably be in it's own category as any user (included the "guest" user in the blacklight sense should be allowed to see the links for Reading Room Requests.